numpy/core/numeric.py

```diff
@@ -2315,6 +2315,10 @@ def isclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
     Unlike the builtin `math.isclose`, the above equation is not symmetric
     in `a` and `b`, so that `isclose(a, b)` might be different from
     `isclose(b, a)` in some rare cases.
+    
+    **When comparing numbers that are significantly less than one,** `atol`
+    should be proportionately reduced to avoid false positives (see the above
```

Comment:
>... false positives ...

Unless they are _true positives_ because that is what you desire.

Reply:
I think it's fine as it is.